app/models/ability.rb

```diff
@@ -24,7 +24,7 @@ def image_editor_permissions
     can [:file_manager, :save_structure], MultiVolumeWork
     can [:create, :read, :edit, :update, :publish, :download], FileSet
     can [:create, :read, :edit, :update, :publish], Collection
-
+    can [:manage], ScannedResource
```

Comment:
I don't think we want this — I think image editors shouldn't be allowed to delete ScannedResources that have ARKs assigned, for example.